exchanges/alphapoint/alphapoint_wrapper.go

```diff
@@ -125,9 +125,11 @@ func (a *Alphapoint) SubmitOrder(p pair.CurrencyPair, side exchange.OrderSide, o
 
 // ModifyOrder will allow of changing orderbook placement and limit to
 // market conversion
-func (a *Alphapoint) ModifyOrder(orderID int64, action exchange.ModifyOrder) (int64, error) {
+func (a *Alphapoint) ModifyOrder(action exchange.ModifyOrder) exchange.ModifyOrderResponse {
 	// return a.ModifyExistingOrder(p.Pair().String(), orderID, action)
```

Comment:
Any reason for not implementing this?
Can remove the comment on ln129

Reply:
Missed it